spyder/plugins/editor/widgets/codeeditor.py

```diff
@@ -888,7 +890,7 @@ def process_signatures(self, params):
                         parameter['label']))

```

Comment:
All the code above is already performing styling, this should probably be moved to the mixin.

Thoughts?

Reply:
Agreed. That was added by @andfoy to (temporarily) make things work with tooltips, but there's no need to have it here.